src/python/pants/backend/python/goals/coverage_py.py

```diff
@@ -525,11 +568,12 @@ async def generate_coverage_reports(
 def _get_coverage_report(
     output_dir: PurePath,
     report_type: CoverageReportType,
+    fail_build: bool,
```

Comment:
It's odd that this is verby. Should it be `(build_)failed` instead?

Reply:
I read it as an instruction to fail the enclosing build.

Reply:
I.e., as an imperative.

Reply:
I'll just change it to "coverage_insufficient", which is more descriptive, and leaves the decision on what to do with that fact to the generic goal.